### DISCUSSION TITLE
Can't ssh to new servers provisioned from a CloudFormation stack update

### DISCUSSION THREAD

**Discussion by yalegria** [mozilla/hubs#4044] [Hubs Cloud Questions] [answered]
I scaled up my app servers to 2. I am not able to ssh to the new app server using the same key and security group that is automatically provisioned by the CloudFormation template. I used to use user ubuntu to login to the EC2 instance. Now i am getting connection refused, permission denied errors when I tried to ssh.

**Comment by yalegria** [comment]
From what I am reading from the documentation, accessing the EC2 server will require 2FA authentication, that needs to be setup on the Admin section. What about if I don't have access to Admin because my site is broken? I need to ssh to the server(s) so I can check the logs and services. Help!

**Reply by johnshaughnessy** [reply]
> What about if I don't have access to Admin because my site is broken? 

Unfortunately if you have not been able to access the admin panel via the admin email you configured when creating the stack, you will not be able to set up 2FA and so will not be able to `ssh` into the machine. There's no workaround for this -- the only fix I know of is to tear down your stack and start fresh.

**Comment by johnshaughnessy** [comment, marked as answer]
Marking https://github.com/mozilla/hubs/discussions/4044#discussioncomment-554495 as answer
